src/twisted/newsfragments/9300.bugfix

```diff
@@ -0,0 +1 @@
+Fixed a race condition in twisted.web.wsgi.WSGIResource which would emit "Unhandled error in Deferred" log messages for wsgi apps.
```

Comment:
Not sure if this is better. The idea is that the release note should help someone decide if they were previously affected by this error or not.

A race condition in twisted.web.wsgi.WSGIResource caused by writes being done after the request was finished no longer occur. The condition was triggering  "Unhandled error in Deferred" type of errors.

Reply:
Done.